src/Terrain/MapFog.java

```diff
@@ -9,7 +9,7 @@
 import Engine.XYCoord;
 import Units.Unit;
 
-public class MapWindow extends GameMap
+public class MapFog extends GameMap
```

Comment:
Not super stoked on `MapFog` as a name, since this object contains much more than just fog tracking. Are there any alternatives that would be clearer on the purpose of this class?